lib/helpers.c

```diff
@@ -224,5 +226,6 @@ struct file_entry cpy_file_entry(struct file_entry fe) {
     copied_fe.comment_after_value = strdup(fe.comment_after_value);
   else
     copied_fe.comment_after_value = NULL;  
+  copied_fe.line_number = 0;
```

Comment:
Yes, it is an error that it is not initialized. But I think that it should be:
copied_fe.line_number =  fe.line_number